### DISCUSSION TITLE
Make ip_range and subnet_ip_range configurable to be able to add dditional subnet for vswitch

### DISCUSSION THREAD

**Discussion by quorak** [vitobotta/hetzner-k3s#242] [Ideas]
I would like to connect a dedicated machine via the private networks. Currently we setup both rangrs with the same parameter. If we make this separate, we have more usecases. Thanks for considering 

https://github.com/vitobotta/hetzner-k3s/blob/62ab7a704fbeba283b1075257d3bf9dd560f74c6/src/hetzner/network/create.cr#L44C28-L44C28

**Comment by vitobotta** [comment]
Can you clarify what you are suggesting I change? BTW there is another discussion about dedicated servers https://github.com/vitobotta/hetzner-k3s/discussions/231
